src/Microsoft.Health.SqlServer/Configs/SqlServerDataStoreConfiguration.cs

```diff
@@ -46,5 +46,10 @@ public class SqlServerDataStoreConfiguration
         /// If set, Instructs the service to terminate when its schema reaches the specified version.
         /// </summary>
         public int? TerminateWhenSchemaVersionUpdatedTo { get; set; }
+
+        /// <summary>
+        /// If set, the app ID of the user assigned managed identity to use when connecting to SQL, if AUthenticationType == ManagedIdentity.
+        /// </summary>
+        public string UserAssignedManagedIdentityAppId { get; set; }
```

Comment:
Do you think it makes sense to make this align with the other setting? Not sure it's a huge deal. 

Also, `AUthenticationType` -> `AuthenticationType`